psi4/driver/procrouting/proc.py

```diff
@@ -1253,7 +1253,7 @@ def scf_helper(name, post_scf=True, **kwargs):
     # Grab a few kwargs
     use_c1 = kwargs.get('use_c1', False)
     scf_molecule = kwargs.get('molecule', core.get_active_molecule())
-    read_orbitals = core.get_option('SCF', 'GUESS') == "READ"
+    read_orbitals = core.get_option('SCF', 'GUESS') is "READ"
```

Comment:
I've had a change of opinion on this one after re-reading. equality with `is` should only be used for the singletons like `True`, `False`, `None`. This is a plain string so `==` appropriate. https://stackoverflow.com/a/2988117

Nevertheless, I don't want to run CI again, and it'll function as-is, so I favor merging (thanks for all your patience with the rebases), then this can be returned to `==` in another PR.

```
>>> def read():
...     return "READ"
... 
>>> read() == "READ"
True
>>> read() is "READ"
<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
True
>>> read() == "NOREAD"
False
>>> read() is "NOREAD"
<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
False
```